src/lambda_handler.py

```diff
@@ -36,50 +40,159 @@ def task_default(event):
     return result
 
 
-def task_cowsay(event) -> dict[str, Union[Optional[str], bool]]:
-    """Generate an output message using the provided information."""
-    result: dict[str, Union[Optional[str], bool]] = {"message": None, "success": True}
-
-    character: str = event.get("character", "tux")
-    if character not in cowsay.characters.CHARS.keys():
-        error_msg = 'Character "%s" is not valid.'
-        logging.error(error_msg, character)
-        failed_task(result, error_msg % character)
-    else:
-        contents: str = event.get("contents", "Hello from AWS Lambda!")
-        logger.info(
-            'Creating output using "%s" with contents "%s"', character, contents
-        )
-        result["message"] = cowsay.get_output_string(character, contents)
-
-    return result
-
-
-def task_divide(event) -> dict[str, Union[Optional[float], bool]]:
-    """Divide one number by another and provide the result."""
-    result: dict[str, Union[Optional[float], bool]] = {"result": None, "success": True}
-    numerator: str = event.get("numerator", None)
-    denominator: str = event.get("denominator", None)
+def validate_event_data(event: Dict[str, Any]) -> EventValidation:
+    """Validate the event data and return a tuple containing the validated event, a boolean result (True if valid, False if invalid), and a list of error message strings."""
+    result = True
+    errors = []
 
-    if denominator is None or numerator is None:
-        error_msg = "Request must include both a numerator and a denominator."
-        logging.error(error_msg)
-        failed_task(result, error_msg)
+    # Check that expiration_days can be interpreted as a strictly positive
+    # integer.
+    if "expiration_days" not in event:
+        errors.append('Missing required key "expiration_days" in event.')
+    elif not event["expiration_days"]:
+        errors.append('"account_ids" must be non-null.')
```

Comment:
Looks like you copy/pasted this from elsewhere, but didn't fully update it.  Is this what you meant?
```suggestion
    elif not event["expiration_days"]:
        errors.append('"expiration_days" must be non-null.')
```